recipes_source/bundled_inputs.rst

```diff
@@ -0,0 +1,193 @@
+(beta) Bundling inputs to PyTorch Models
+==================================================================
+
+**Author**: `Jacob Szwejbka <https://github.com/JacobSzwejbka>`_
+
+Introduction
+------------
+
+This tutorial introduces the steps to use PyTorch's utility to bundle example or trivial inputs directly into your TorchScript Module.
+
+The interface of the model is not changed (other than adding a few methods), so it can still be safely deployed to production. The advantage of this standardized interface is that tools that run models can use it instead of having some sort of external file (or worse, document) that tells you how to run the model properly.
+
+Common case, bundling an input to a model that only uses 'forward' for inference
+-------------------
+1. **Prepare model**: Convert your model to TorchScript through either tracing or scripting
+
+.. code:: python
+
+    import torch
+    import torch.jit
+    import torch.utils
+    import torch.utils.bundled_inputs
+
+    class Net(nn.Module):
+        def __init__(self):
+            super(Net, self).__init__()
+            self.lin = nn.Linear(10, 1)
+
+        def forward(self, x):
+            return self.lin(x)
+
+    model = Net()
+    scripted_module = torch.jit.script(model)
+
+2. **Create example input and attach to model**
+
+.. code:: python
+
+    # For each method create a list of inputs and each input is a tuple of arguments
+    input = [(torch.zeros(1,10),)]
+
+    # Create model with bundled inputs, if type(input) is list then the input is bundled to 'forward'
+    bundled_model = bundle_inputs(scripted_module, input)
+
+
+3. **Retrieve inputs and run model on them**
+
+.. code:: python
+    inputs = bundled_model.get_all_bundled_inputs()
+
+    print(bundled_model(*inputs[0]))
```

Comment:
Just FYI Netlify render might be old [here](https://deploy-preview-1524--pytorch-tutorials-preview.netlify.app/recipes/bundled_inputs?utm_source=github&utm_campaign=bot_dp)

But this code part is not visible in the netlify render of the page, please check if there is any whitespace/tab or something thats missing.